pkg/cfaws/assumer_aws_sso.go

```diff
@@ -54,17 +54,17 @@ func (c *CFSharedConfig) SSOLogin(ctx context.Context) (aws.Credentials, error)
 		return aws.Credentials{}, err
 	}
 	cachedToken := GetValidCachedToken(ssoTokenKey)
-	newToken := false
+	// newToken := false
```

Comment:
Looks like this was commented out by accident

Reply:
Not by accident. to update the descriptions I needed to redo a cache of the token to update it. 
I tried to make a lite version (so that we are not calling storessokey every time) that just updates the description of the key but the Ring package doesn't allow for updating keys just overriding.